tests/Integration/Behaviour/Features/Scenario/Search/add_alias.feature

```diff
@@ -0,0 +1,47 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s alias --tags add
+@restore-aliases-before-feature
+@clear-cache-before-feature
+@clear-cache-after-feature
+@add
+Feature: Add basic alias from Back Office (BO)
+  As a BO user
+  I need to be able to add new alias with basic information from the BO
+
+  Scenario: I add an alias with basic information
+    Given following aliases should exist:
+      | id reference | alias  | search |
+      | alias1       | bloose | blouse |
+      | alias2       | blues  | blouse |
+    When I add alias with following information:
+      | alias   | large |
+      | search  | big   |
+    Then following aliases should exist:
+      | id reference | alias  | search |
+      | alias1       | bloose | blouse |
+      | alias2       | blues  | blouse |
+      | alias3       | large  | big    |
+
+  Scenario: I add a multiple aliases with basic information
+    When I add alias with following information:
+      | alias   | bliu,blu |
+      | search  | blue     |
+    Then following aliases should exist:
+      | id reference | alias  | search |
+      | alias1       | bloose | blouse |
+      | alias2       | blues  | blouse |
+      | alias3       | large  | big    |
+      | alias5       | bliu   | blue   |
+      | alias6       | blu    | blue   |
+
+  Scenario: I add a multiple aliases with duplicate search field
+    When I add alias with following information:
+      | alias   | bliu,sapphire |
+      | search  | blue          |
+    Then following aliases should exist:
+      | id reference | alias    | search |
+      | alias1       | bloose   | blouse |
+      | alias2       | blues    | blouse |
+      | alias3       | large    | big    |
+      | alias5       | bliu     | blue   |
+      | alias6       | blu      | blue   |
+      | alias7       | sapphire | blue   |
```

Comment:
Add another step here to see what happens when you remove some of aliases from existing search (I guess it should just append the "blah" but not delete other aliases that existed for that term (bliu and blu))  something like this:
```
When I add alias with following information:
      | alias   | blah |
      | search  | blue          |
    Then following aliases should exist:
      | id reference | alias    | search |
      | alias1       | bloose   | blouse |
      | alias2       | blues    | blouse |
      | alias3       | large    | big    |
      | alias5       | bliu     | blue   |
      | alias6       | blu      | blue   |
      | alias7       | sapphire | blue   |
      | alias8       | blah | blue   |
```
add a scenario's when we leave searchTerm and/or alias empty string (im not sure what happens then).
also a scenario where

Reply:
Also a scenario's when we leave searchTerm and/or alias empty string (im not sure what happens then).